feat: Presentation support & IIW credential support [DEV-1883]

jay-dee7 opened the pull request.

jay-dee7 committed:
feat: Import credentials from PDFs

This PR adds a feature to read data from QR Codes embedded in PDF files.
Then, this data is chained to a credential issuing service to retrieve a
credential.

- [x] Parse QR Codes from PDF
- [ ] Request a Credential using the parsed data

jay-dee7 committed:
fix: IIW logo for Dark mode

DaevMithran committed:
feat: Add veramo presentation support

DaevMithran committed:
feat: Add createPresentation method && VeramoSubjectId

jay-dee7 committed:
add: Select credentials for presentation

jay-dee7 committed:
Merge branch 'read-qr-code-from-input' into presentationSupport

DaevMithran committed:
feat: Add QR for presentation

DaevMithran committed:
feat: Add DetailsPopup component

DaevMithran committed:
feat: Issue IIW pass with eventId

jay-dee7 committed:
fix: Import credential button and credentials order

DaevMithran committed:
feat: export&import holder dids

DaevMithran committed:
Merge branch 'IIW-demo' of github.com:cheqd/wallet into IIW-demo

ankurdotb committed:
Merge branch 'cheqd' into IIW-demo

ankurdotb committed:
Update package-lock.json

jay-dee7 committed:
fix: Broken builds

DaevMithran committed:
feat: Issue IIW pass with eventId (#111)

Questions
- I'm currently passing the eventId of IIW pass in the parameter itself
- I'm can't pass in body as I'm not able to use await request.json()
twice in the code, Facing an internal server error
- @jay-dee7 said this maybe due to type-casting, But not able to resolve
that yet

DaevMithran committed:
fix: Minor issues

ankurdotb commented:
Task linked: [DEV-1883 Creating a Verifiable Presentation from cheqd wallet](https://app.clickup.com/t/6600954/DEV-1883)

DaevMithran commented:
This PR also include [DEV-1914]
Steps to test, logout of the wallet -> Login -> getCredential
All the credentials should have the same DID

ankurdotb committed:
npm dedupe

ankurdotb committed:
Making QR code larger

ankurdotb committed:
Update DetailsPopup.tsx

ankurdotb committed:
Fix typo in setCredentialMode

ankurdotb committed:
Update issuer.ts

ankurdotb committed:
Update issuer.ts

ankurdotb committed:
Update Identity.tsx

DaevMithran committed:
feat: Remove social account for IIW

DaevMithran committed:
fix: Fix QR handling && IIW flow [DEV-1921] (#116)

1. IIW credential is no longer dependant on a social account
2. Importing same file did not trigger a credential before, that has
been fixed now

ankurdotb committed:
Add Github option

DaevMithran committed:
fix: Provide service name

DaevMithran committed:
Merge branch 'cheqd' into presentationSupport

DaevMithran committed:
feat: Add verifyPresentation Support

DaevMithran committed:
feat: Update formatted view

DaevMithran committed:
Merge branch 'cheqd' into presentationSupport

jay-dee7 committed:
fix: Modal for Presentation

DaevMithran committed:
feat: Add url for QR

DaevMithran committed:
Merge branch 'url-shortener' into presentationSupport

jay-dee7 committed:
fix: Modal title highlights

jay-dee7 committed:
Merge branch 'presentationSupport' of github.com:cheqd/wallet into presentationSupport

rosspower11 reviewed (approved):
As agreed with @Eengineer1 @jay-dee7 @DaevMithran  on a demo walkthrough, we'll merge this one in to main. Remaining items for the demo are to include schema as a resource and a logo as a resource, which will be taken in a follow-up PR
